tests/e2e/e2e_suite_test.go

```diff
@@ -138,3 +145,27 @@ var _ = BeforeSuite(func() {
 		Expect(err).ToNot(HaveOccurred())
 	})
 })
+
+var _ = AfterSuite(func() {
+
+	if os.Getenv("NO_TEARDOWN") != "" {
+		Skip("No teardown")
+	}
+
+	err := exec.Command("system-tools", "remove", "--force", "-c", kubeconfig).Run()
```

Comment:
this doesnt exists on my system :o

Reply:
yep, In the long run I plan to move it to a kube pod handling deletion.. just trying it out locally at this point